tensorflow/contrib/grid_rnn/python/kernel_tests/grid_rnn_test.py

```diff
@@ -0,0 +1,324 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Tests for GridRNN cells."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+
+class GridRNNCellTest(tf.test.TestCase):
+
+  def testGrid2BasicLSTMCell(self):
+    with self.test_session() as sess:
+      with tf.variable_scope('root', initializer=tf.constant_initializer(0.2)) as root_scope:
+        x = tf.zeros([1, 3])
+        m = tf.zeros([1, 8])
+        cell = tf.contrib.grid_rnn.Grid2BasicLSTMCell(2, input_size=3)
+        self.assertEqual(cell.output_size, 2)
```

Comment:
we're getting rid of both input_size and output_size; output_size is not needed and input_size can be replaced with using get_shape() on the input inside the cell.  please remove these from tests and update the cell accordingly.